keyboards/svalboard/keymaps/vial/vial.json

```
@@ -45,6 +45,26 @@
         "shortName": "Mouse\nKey\nTimer",
         "title": "Toggle between various settings of the mouse keys timer.",
```

cycle is the appropriate word here
```suggestion
        "title": "Cycle between various settings of the mouse keys timer.",
```